relay-server/src/services/spooler/mod.rs

```diff
@@ -1569,26 +1569,25 @@ mod tests {
             "buffer.writes:1|c",
             "buffer.envelopes_written:3|c",
             "buffer.envelopes_disk_count:3|g",
-            "buffer.disk_size:1031|h",
+            "buffer.disk_size:24576|h",
```

Comment:
This is pretty far off. But the relative error should be lower for larger disk buffers.